# Analyzing CRISPR screen data

This repository contains material for the HMS Genetics Bootcamp held on January 13, 2026.

## Why study CRISPR screening?
CRISPR technologies have revolutionized modern genetics by enabling precise, programmable manipulation of the genome. High-throughput CRISPR platforms support 
systematic gene perturbations, including knockout, knockdown, and overexpression. Coupled with next generation sequencing, these approaches enable quantitative phenotypic readouts at scale. Next generation genome editing technologies such as base and prime editing extend this framework to single nucleotide resolution, providing deeper insights into the genetic variants that shape complex traits. This workshop introduces the design principles and uses cases of CRISPR screening, and provides hands-on experience analyzing screen data, interpreting genetic effects, and generating testable hypotheses. 

In this workshop, students will discuss screens performed by the lab and in the scientific literature. Finally, students will participate in two dry lab sessions where they will 1) get exposed to the computational 
tools that can inform target selection and library design and 2) explore gene targeting and variant scanning CRISPR screening datasets.

## Course schedule

- 8:50 am-9:00 am: Meet in the VSC (NRB) Lobby 
- 9:00 am-9:45 am: CRISPR screening lecture (VSC 230)
- 9:45 am-10:00 am: Q+A (VSC 230)
- 10:00 - 12:00 pm: Morning lab: CRISPR screen design (VSC 230)
- 12:00 -1:00 pm: Lunch and mini-presentations from lab members (VSC 230)
- 1:00 - 3:30 pm: Afternoon lab: Exploring a real CRISPR screen (VSC 230)
- - Test
- 3:30 - 4:00 pm: Wrap-up (VSC 230)


## Materials

Presentation slides for the lecture, morning, and afternoon labs are included as downloadable files in this GitHub repository.

Please consult the following link for a tutorial on the prime editing screening done in our preprint for the afternoon lab (Zhou et al. 2025). 
https://colab.research.google.com/github/pzhou1729/2026_CRISPRscreen_GeneticsBootcamp/blob/main/LDLR_PE_tutorial.ipynb#scrollTo=b20a9f4f-fef1-40cd-8b79-b14e7108f58e

## Recommended reading

A new era in functional genomics screens (https://www.nature.com/articles/s41576-021-00409-w)

Joint genotypic and phenotypic outcome modeling improves base editing variant effect quantification
(https://www.nature.com/articles/s41588-024-01726-6)

LDLR variant classification through activity-normalized prime editing screening
(https://www.biorxiv.org/content/10.64898/2025.12.16.694467v1)
